tests/pom.xml

```diff
@@ -215,6 +218,21 @@
       <version>4.2.12</version>
```

Comment:
org.apache.felix.webconsole 4.2.14 is available on Maven Central, the following dependencies are not needed: org.osgi.service.component, org.osgi.util.promise, org.osgi.util.function